### ISSUE TITLE
bowels revamp

### ISSUE THREAD

**Issue by SSgamingYT** [nonamecrackers2/crackers-wither-storm-mod#480]
I got an idea for the bowels.
perhaps you could make the bowels be literally inside the wither storm, so basically it will stop moving and generate the bowels inside of itself, but only when a player enters it. just a small suggestion i thought of, it would be nice.

bonus: maybe make the wither storm blow up and flesh blocks that slowly turn into normal blocks fall out, and the bowels itself will fall as a block cluster entity, allowing you to make it more like story mode.
TL;DR: make chunks the wither storm and the bowels(continuing my suggestion) turn into block clusters that fall to the ground while flying out from the storm(it would be cool)

**Comment by watersofnazareth** [comment]
This would be nearly impossible to do but also #184

**Comment by personmhmyes** [comment]
with the bowels idea instead of it generating itself inside of itself it could work the same of get through the hole wise but it will be randomly generated with a set distance 2-5 pathways and a command block chamber to make it intresting then other bits like pillers and withered trees etc giving more variation

**Comment by nonamecrackers2** [comment]
It's a separate dimension for a reason. You wouldn't have the special fog and sound effects, the blocks would need to move with the Wither Storm, I also don't think they could even fit inside the WS model, and more.

**Comment by SSgamingYT** [comment]
@nonamecrackers2 if you implement the gravity thing, you can make the bowels sideways instead as long as the wither storm stays still it should work

**Comment by SSgamingYT** [comment]
but the fog and sounds are understandable. in bedrock you can easily just use /fog but this is java
